elastic/kibana
# [ResponseOps][Task manager] Enhance release claim failure message

Resolves https://github.com/elastic/kibana/issues/175987

## Summary

This PR fixes the release claim failure log message logging `[object Object]` that is the result of a 409 saved object conflict error shown below.

Saved object error:
```
{
  type: 'type',
   id: 'id',
   error: {
     statusCode: 409,
     error: 'Conflict',
     message: 'Saved object [type/id] conflict',
  },
}

```

Log message:
`[Task Runner] Task ... failed to release claim after failure: [object Object]`


### Checklist


### To verify
This error is hard to replicate locally, so I am sorry for the weird verification steps.

1. Run the telemetry test server
```
node scripts/functional_tests_server.js --config x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/telemetry/config.ts
```
3. Make a change in the server code to trigger the test server to reload. I just added a comment to `x-pack/plugins/task_manager/server/task_running/task_runner.ts`
4. Search for the following log, and verify that the message does not include `[object Object]` 
```
[ERROR][plugins.taskManager] [Task Runner] Task ML:saved-objects-sync-task failed to release claim after failure
```
